### PR TITLE
Proposing to add a lot of entries from kowith337

### PR DESCRIPTION
**Pull request by DandelionSprout**
Okay, here's my plan:

I am planning to make pull requests to uBlock Origin and Nano Adblocker to add some additional source links for some lists (such as for their Lithuanian and Norwegian lists), and I've taken note that there's now quite a few Thai adblock lists out there (as can be seen on https://filterlists.com/), but none of them have a particularly wide coverage of Thai sites.

So in order to create a properly Thai list for which I could make a case for it to be included natively in uBlock Origin, I hereby suggest that you can add some Thai entries from @kowith337's lists, especially from [Thai Paranoid Block](https://github.com/kowith337/ThaiParanoidBlock/blob/master/ThaiParanoid.txt) and [Thai Consent Hosts](https://github.com/kowith337/ThaiConsentHosts/blob/master/ThaiConsentHosts.txt).

Since he hasn't added a licence to his lists yet, I suppose I'll have to ask him about it here through this comment: @kowith337, are you okay with having entries from your lists be included in EasyList Thailand?

I've made this pull request to get this project off the ground, but this is a one-off thing I'm doing. In the future I suppose that you could work together or to find other local enthusiasts to help you, and I wish you good luck with it.

### CODE REVIEW

**Comment by gluons** [comment]
Filters shoud not be added into `./subscription/easylist-thailand.txt` file.  
It's generated from my script.

I've separated filters to individual category. It's easier to maintain.  
You should place filters in the correct category in [`./categories`](https://github.com/easylist-thailand/easylist-thailand/tree/master/categories) directory.

I'm glad if @kowith337 can help us to maintain EasyList Thailand.  
I'm quite busy.

**Comment by DandelionSprout** [comment]
>I've separated filters to individual category. It's easier to maintain.
You should place filters in the correct category in ./categories directory.

Aaaaah, okay. I'll attempt to correct my mistake, then. 🙂

**Comment by gluons** [comment]
OK. I'll wait for @kowith337 permission before merge this PR.  
Thank you very much.

**Comment by kowith337** [comment]
Sorry for late respond, but thanks for attention.
My filterlist is free to include or modify to other lists, but I didn't have a proper categorize (only divide blocking type like DOM, sites, requests, scriptlets, etc, not the type of website) maybe need to create another pull or divide those rules into each proper categories.

First of all,
- My lists may not have much rules for some type of site, especially about anime, movies, series, adult and other like (due to local law and most of site in these categories are putting ads too much that I cannot putting me into the risk)
- Some sites are strict blocking by me, e.g.
  - mobile badvertisers domains that can auto subscribe wasteful services without any confirmation from user's side (mostly reported on Pantip MBK forum)
  - known third-party ads and trackings (none of any allowance, even in first party sites or for an advance users who works about websites and networking)
  - SNS that tracking users through embed/widget (Facebook Connect, Twitter Widget, Google+1, etc.)

In other words, I'm not familiar to any automation or generator, I'm still need to learn or start from the beginning about how it can do, how to use and how it works, etc.

**Comment by DandelionSprout** [comment]
As far as my plan goes at the moment, this pull request would place the borrowed entries into a special "kowith337" category. Any future entries that'd be added by you (or by anyone else) would've been added to the regular categories at https://github.com/easylist-thailand/easylist-thailand/tree/master/categories.

If you by "automation or generator" think of auto-adding entries, then it's not mandatory to do so (although it'd of course be pretty neat) unless one works with e.g. malware domains, since those domains have quick turnover rates. After all, every single one of the entries in [my Norwegian list](https://github.com/DandelionSprout/adfilt/raw/master/NorwegianList.txt) were discovered and added to that list manually.

>My filterlist is free to include or modify to other lists

Ah, okay. I'll treat that as that gluons can accept this pull when (s)he wakes up this morning.

**Comment by kowith337** [comment]
Thanks again, I will check this out and may help to update the proper rearrange from my lists later if I have some chance of free time.

PS: @DandelionSprout did you mind about I will remove the `kowith337.txt` file later and then let me update for each categories or not?

**Comment by gluons** [comment]
**Notice:** I'll run the generator to combine all filters into `subscription/easylist-thailand.txt` ASAP.

**Comment by DandelionSprout** [comment]
@kowith337
>do you mind about I will remove the kowith337.txt file later and then let me update for each categories or not?

 Of course I won't mind. I'll let you guys stake out the course of this list from now on. 🙂

**Comment by DandelionSprout** [comment]
>I'll run the generator to combine all filters into subscription/easylist-thailand.txt ASAP.

It doesn't seem to have been completed yet, since the new entries still aren't visible in https://github.com/easylist-thailand/easylist-thailand/blob/master/subscription/easylist-thailand.txt at the time of writing.

**Comment by gluons** [comment]
Done.
